tests/e2e/app/authn-webauthn/autoscript.js

```diff
@@ -61,15 +56,20 @@
       }),
       rxjs.operators.delay(delay),
       rxMergeMap(
-        (step) => {
+        async (step) => {
           const webAuthnStep = forgerock.FRWebAuthn.getWebAuthnStepType(step);
           if (webAuthnStep === 2) {
             console.log('WebAuthn step is registration');
           } else {
             throw new Error('WebAuthn step is incorrectly identified');
           }
           console.log('Handle WebAuthn Registration');
-          return forgerock.FRWebAuthn.register(step);
+          try {
+            step = await forgerock.FRWebAuthn.register(step);
+          } catch (err) {
+            console.log(err);
+          }
+          return step;
```

Comment:
Adding some try-catches to help observe errors and their submission to AM for testing.